### PR TITLE
Add a KUBERNETES_NODE_* section to build kubelet/kube-proxy for windows

### PR DESCRIPTION
**Pull request by brendandburns**
@saad-ali @luxas @pires

### CODE REVIEW

**Comment by saad-ali** [comment]
Is the cross build failure related or a just a flake? Also is this cherry-picking an existing PR?

File: hack/lib/golang.sh [hunk 1]
```diff
@@ -67,6 +81,19 @@ else
   fi
   readonly KUBE_SERVER_PLATFORMS
 
+  # The node platforms we build for
+  KUBE_NODE_PLATFORMS=(
+    linux/amd64
+    linux/arm
+    linux/arm64
+    linux/s390x
```
Comment (ixdy) [inline hack/lib/golang.sh]: s390x isn't supported on k8s 1.5. This needs to be removed.

**Comment by luxas** [comment]
Yes, I think that once you've fixed the `s390x` thing, the crossbuild should be working.

Having the crossbuild green is crucial, otherwise we're not able to cut any releases.

**Comment by brendandburns** [comment]
@luxas @ixdy @saad-ali cross build is now clean, thanks for the pointer to the fix.

**Review by pires — APPROVED** [review]

**Comment by pires** [comment]
@saad-ali what's the difference between approved and lgtm labels? Seems the bot wants the approved one again.

**Comment by luxas** [comment]
@pires It might be a bug as well; I've seen my PRs been automatically approved because I'm in the OWNERS file. I think the same logic should apply here... but not totally sure if the SQ's behaviour was just a bug or not...

Anyway, seems like it got retriggered/reapproved now

**Comment by pires** [comment]
Thanks @luxas.

**Pull request merged**